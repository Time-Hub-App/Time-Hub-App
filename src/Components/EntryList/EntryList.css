.entrylist {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  margin: 20px;
  gap: 20px;
}

.entry {
  border: aliceblue 2px solid;
  width: 40vw;
}

.entry h3 {
  margin: 10px 10px;
}

.entry h3:hover {
  cursor: pointer;
  text-decoration: underline;
}
