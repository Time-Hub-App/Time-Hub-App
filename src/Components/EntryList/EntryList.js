import React from 'react';

export default function EntryList() {
  return <div></div>;
}
