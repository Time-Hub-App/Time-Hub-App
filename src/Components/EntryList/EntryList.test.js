import { render } from '@testing-library/react';
import { useState } from 'react';
import { MemoryRouter } from 'react-router-dom';
import EntryList from './EntryList';


it('entry list render', async () => {
  const entries = [{
    id: 1,
    text: 'meowmeow',
    emotion: 'Joy'
  }];
  const { container } = render(
    <MemoryRouter>
      <EntryList entries={entries}/>
    </MemoryRouter>

  );

  expect(container).toMatchSnapshot();
});