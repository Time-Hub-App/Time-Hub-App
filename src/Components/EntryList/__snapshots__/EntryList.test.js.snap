// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entry list render 1`] = `
<div>
  <div
    class="entrylist"
  >
    <div
      class="entry"
    >
      <p>
        meowmeow
      </p>
      <p>
        Joy
      </p>
      <h3>
        Delete Entry
      </h3>
      <h3>
        Update Entry
      </h3>
    </div>
  </div>
</div>
`;
